doc/lspconfig.txt

```diff
@@ -1,242 +1,659 @@
-language servers you have installed on your system. Language server
-configurations are community-maintained.
 ==============================================================================
-2. LSP overview                                       *lspconfig-lsp-overview*
+LSP OVERVIEW                                                     *lspconfig-lsp*
 
+Neovim supports the Language Server Protocol (LSP) via the built-in language
+server client. LSP facilitates many features, some of which include:
 
-Neovim supports the Language Server Protocol (LSP), which means it acts as a
-client to language servers and includes a Lua framework `vim.lsp` for building
-enhanced LSP tools. LSP facilitates features like:
 - go-to-definition
 - find-references
 - hover
 - completion
 - rename
 - format
 - refactor
-Neovim provides an interface for all of these features, and the language server
-client is designed to be highly extensible to allow plugins to integrate
-language server features which are not yet present in Neovim core such as [**auto**-completion](https://github.com/neovim/nvim-lspconfig/wiki/Autocompletion)
-(as opposed to manual completion with omnifunc) and [snippet integration](https://github.com/neovim/nvim-lspconfig/wiki/Snippets).
-These features are not implemented in this repo, but in Neovim core. See `:help
-lsp` for more details.
+
+These features are implemented in neovim core, not lspconfig. See `:help lsp`
+for more details.
+
+NOTE: Feature availability depends on the implementation details of the
+server. that servers may implement only a subset of these features. Always
+consult the server documentation before filing a bug report on a missing
+feature.
+
 ==============================================================================
-3. Install                                                 *lspconfig-install*
+QUICKSTART                                                *lspconfig-quickstart*
 
+- ensure the server is installed and executable from the command line
 
-* Requires [Neovim v0.5.0](https://github.com/neovim/neovim/releases/tag/v0.5.0) or [Nightly](https://github.com/neovim/neovim/releases/tag/nightly). Update Neovim and nvim-lspconfig before reporting an issue.
-* Install nvim-lspconfig like any other Vim plugin, e.g. with [vim-plug](https://github.com/junegunn/vim-plug):
+- enable the server in your neovim configuration (lua example):
+>
+    require'lspconfig'.clangd.setup{}
+<
+- create a new project, ensure it contains a root marker which matches the
+  server requirements specified in `servers.md`.
+
+- open a file within that project, such as `main.c`.
 
-    ```vim
-    Plug 'neovim/nvim-lspconfig'
-    ```
 ==============================================================================
-4. Quickstart                                           *lspconfig-quickstart*
+SETUP                                                          *lspconfig-setup*
```

Comment:
I feel like this makes it sound as if there's a function called `lspconfig.setup()`.